_posts/2018-09-19-cockpit-178.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 178
+author: mvo
+date: 2018-09-19
+tags: cockpit linux
+slug: cockpit-178
+category: release
+summary: Fixed crashers and dropped KubeVirt support
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 178.
+
+### KubeVirt support has been removed
+
+Cockpit shipped `cockpit-kubernetes` for managing Kubernetes and OpenShift deployments in previous releases. Since its introduction, important points have come to light:
```

Comment:
As we don't drop the whole package yet, maybe s/shipped in previous releases/ships/?